packages/ripple-iso-crypto/src/internal/wrapCryptoCreateHash.ts

```diff
@@ -0,0 +1,25 @@
+import { createHash } from 'crypto'
+import { HashFn, Input } from './index'
+import normInput from './normInput'
+
+export default function wrapCryptoCreateHash(
+  type: string,
+  fn: typeof createHash,
+) {
+  const hashFn = ((input: Input) => {
+    return new Uint8Array(fn(type).update(normInput(input)).digest().buffer)
```

Comment:
TODO
just return the Buffer ... 
for all intents and purposes it IS a Uint8Array